docs/lib/auth/fragments/flutter/social_signin_web_ui/10_cli_setup.md

```diff
@@ -0,0 +1,22 @@
+In terminal, navigate to your project, run `amplify add auth`, and choose the following options (the last steps are specific to Facebook here but are similar for other providers):
```

Comment:
Which steps are specific to Facebook?  I don't see any below.

Reply:
This was taken directly from the native docs. In any case, I 'm going to remove here.